keanu-project/src/main/java/io/improbable/keanu/model/PredictiveModel.java

```diff
@@ -1,6 +1,6 @@
 package io.improbable.keanu.model;
 
-public interface Model<INPUT, OUTPUT> {
+public interface PredictiveModel<INPUT, OUTPUT> {
```

Comment:
NB: renamed to distinguish this from `ProbabilisticModel` (the two are not related)